src/components/Carousel/Carousel.js

```diff
@@ -0,0 +1,115 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useCallback, useEffect, useState } from 'react';
+
+import LeftArrMobile from '../../assets/svgs/LeftArrMobile.svg';
+import RightArrMobile from '../../assets/svgs/RightArrMobile.svg';
+import {
+  arrowContainer,
+  carousel,
+  img,
+  indicator,
+  indicatorFill,
+  indicators,
+  leftArrowContainer,
+  rightArrowContainer,
+  slider,
+  sliderContainer,
+} from './Carousel.module.scss';
+
+export const Carousel = ({ images }) => {
```

Comment:
dodaj komentarz czym są images, czy to obiekty, ścieżki czy coś innego.
chodzi o to żeby osoba uzywajaca komponentu wiedziala jak przekazać obrazy